### PR TITLE
feat: migrate garden glyphs to shrubs

### PR DESCRIPTION
**Pull request by leandrogrillo**
#### Pull Request Type:

- [ ] Bug :beetle:
- [ ] Chore :gear:
- [x] Feature :rocket:

#### Description of motivation for this Pull Request:

Migrate garden icons to `@backyard/icons` and be available as react components in  `@backyard/shrubs` and  `@backyard/shrubs-react-native`

#### Code snippets

as react components
```js
import { ArrowLeft } from '@backyard-ui/shrubs'
```

as react native componentes
```js
import { ArrowLeft } from '@backyard-ui/shrubs-react-native'
```

### CODE REVIEW

File: packages/icons/src/inspiration.svg [hunk 1]
```diff
@@ -0,0 +1 @@
... 84.703-15.852 48.02-9.14 109.418 19.955 182.728 15.287 38.542 32.656 69.711 41.777 85.04H174.56c-.04 0-.078.012-.117.012-40.84-65.395-101.058-180.724-77.84-262.096zm319.635 15.238c15.715 60.259-9.527 142.215-75.233 244.286h-56.041c62.862-107.097 82.154-196.598 57.362-265.832-17.205-48.045-51.26-73.11-70.788-84.078 22.313 2.518 44.16 8.448 64.305 17.805 43.347 20.083 70.39 49.66 80.395 87.819zM248.382 365.297l-8.173.05c-14.492-23.466-96.426-163.718-64.098-262.353 11.153-34.398 35.553-60.402 72.27-77.265v339.568zm12.864.05V28.302c16.955 9.09 51.046 32.16 67.46 77.609 23.985 66.17 3.561 153.484-60.712 259.396l-6.748.04zm-54.743 48.884c-3.637-5.09-11.915-16.907-22.443-33.447h68.287a7.309 7.309 0 0 0 4.153 1.299 7.305 7.305 0 0 0 4.153-1.3h72.633a1172.918 1172.918 0 0 1-22.755 33.448H206.503zm224.79-295.872c-11.11-42.785-40.922-75.51-88.325-97.635C287.79-5.053 220.76-6.882 163.916 15.79c-43.328 17.186-71.102 45.89-82.212 85.381-14.999 53.022-.37 123.963 43.328 210.626 28.765 56.646 61.304 101.877 68.51 111.708v66.372c0 12.248 10 22.123 22.405 22.123h80.917c12.404 0 22.588-9.875 22.405-22.123v-64.361C412.878 295.971 450.6 192.645 431.293 118.359z" fill="#000" fill-rule="evenodd"/></svg>
```
Comment (andreolvr) [inline packages/icons/src/inspiration.svg]:
```suggestion
<svg width="512" height="512" xmlns="http://www.w3.org/2000/svg"><path d="M297.465 496.563h-80.156c-4.218 0-7.52-3.328-7.52-7.577v-61.89h92.045a7.372 7.372 0 0 0 2.968 4.99c.058.05.122.08.183.127v56.773c0 4.25-3.302 7.577-7.52 7.577zM96.603 105.823c10.073-34.822 34.877-60.297 74.04-75.876 22.196-8.74 46.028-13.56 70.02-14.51-39.39 18.562-65.616 47.086-78.039 84.703-15.852 48.02-9.14 109.418 19.955 182.728 15.287 38.542 32.656 69.711 41.777 85.04H174.56c-.04 0-.078.012-.117.012-40.84-65.395-101.058-180.724-77.84-262.096zm319.635 15.238c15.715 60.259-9.527 142.215-75.233 244.286h-56.041c62.862-107.097 82.154-196.598 57.362-265.832-17.205-48.045-51.26-73.11-70.788-84.078 22.313 2.518 44.16 8.448 64.305 17.805 43.347 20.083 70.39 49.66 80.395 87.819zM248.382 365.297l-8.173.05c-14.492-23.466-96.426-163.718-64.098-262.353 11.153-34.398 35.553-60.402 72.27-77.265v339.568zm12.864.05V28.302c16.955 9.09 51.046 32.16 67.46 77.609 23.985 66.17 3.561 153.484-60.712 259.396l-6.748.04zm-54.743 48.884c-3.637-5.09-11.915-16.907-22.443-33.447h68.287a7.309 7.309 0 0 0 4.153 1.299 7.305 7.305 0 0 0 4.153-1.3h72.633a1172.918 1172.918 0 0 1-22.755 33.448H206.503zm224.79-295.872c-11.11-42.785-40.922-75.51-88.325-97.635C287.79-5.053 220.76-6.882 163.916 15.79c-43.328 17.186-71.102 45.89-82.212 85.381-14.999 53.022-.37 123.963 43.328 210.626 28.765 56.646 61.304 101.877 68.51 111.708v66.372c0 12.248 10 22.123 22.405 22.123h80.917c12.404 0 22.588-9.875 22.405-22.123v-64.361C412.878 295.971 450.6 192.645 431.293 118.359z" fill="#000" fill-rule="evenodd"/></svg>

```
No newline at end of file

File: packages/icons/src/Silhouette.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg width="512" height="512" xmlns="http://www.w3.org/2000/svg"><g fill="#000" fill-rule="nonzero" stroke="#000" stroke-width=".5"><path d="M256.5 323C189.442 323 135 250.648 135 161.586 135 72.524 189.442 0 256.5 0 323.558 0 378 72.352 378 161.414 378 250.476 323.558 323 256.5 323zm0-305.946c-58.071 0-105.254 64.945-105.254 144.532 0 79.587 47.356 144.36 105.254 144.36s105.254-64.773 105.254-144.532c0-79.76-47.183-144.36-105.254-144.36zM9.388 438.49c10.752-32.08 35.669-74.977 93.866-96.975 17.407-6.416 34.644-12.099 52.906-17.232l4.266-1.283 3.414 3.3c26.794 25.664 58.196 39.413 90.281 39.413 32.085 0 62.975-13.382 89.6-38.68l3.242-3.116 4.266 1.1c18.09 5.132 38.4 12.648 57.685 19.981 60.074 22.548 84.82 64.528 95.06 95.875 6.663 20.331 9.222 41.952 7.51 63.428L510.46 512H.855l-.683-8.25c-.969-22.193 2.163-44.373 9.216-65.26zm485.882 55.362c.177-16.033-2.249-31.975-7.167-47.113-12.97-39.963-41.472-68.56-84.65-84.692-17.408-6.6-35.84-13.383-52.224-18.332-29.013 26.214-62.122 39.963-97.278 39.963-35.157 0-68.778-14.115-97.791-40.696-16.384 4.766-31.915 9.899-47.445 15.765-41.472 15.582-69.46 44.363-83.285 85.242-5.432 15.994-8.373 32.841-8.704 49.863H495.27z"/></g></svg>
```
Comment (andreolvr) [inline packages/icons/src/Silhouette.svg]:
```suggestion
<svg width="512" height="512" xmlns="http://www.w3.org/2000/svg"><g fill="#000" fill-rule="nonzero" stroke="#000" stroke-width=".5"><path d="M256.5 323C189.442 323 135 250.648 135 161.586 135 72.524 189.442 0 256.5 0 323.558 0 378 72.352 378 161.414 378 250.476 323.558 323 256.5 323zm0-305.946c-58.071 0-105.254 64.945-105.254 144.532 0 79.587 47.356 144.36 105.254 144.36s105.254-64.773 105.254-144.532c0-79.76-47.183-144.36-105.254-144.36zM9.388 438.49c10.752-32.08 35.669-74.977 93.866-96.975 17.407-6.416 34.644-12.099 52.906-17.232l4.266-1.283 3.414 3.3c26.794 25.664 58.196 39.413 90.281 39.413 32.085 0 62.975-13.382 89.6-38.68l3.242-3.116 4.266 1.1c18.09 5.132 38.4 12.648 57.685 19.981 60.074 22.548 84.82 64.528 95.06 95.875 6.663 20.331 9.222 41.952 7.51 63.428L510.46 512H.855l-.683-8.25c-.969-22.193 2.163-44.373 9.216-65.26zm485.882 55.362c.177-16.033-2.249-31.975-7.167-47.113-12.97-39.963-41.472-68.56-84.65-84.692-17.408-6.6-35.84-13.383-52.224-18.332-29.013 26.214-62.122 39.963-97.278 39.963-35.157 0-68.778-14.115-97.791-40.696-16.384 4.766-31.915 9.899-47.445 15.765-41.472 15.582-69.46 44.363-83.285 85.242-5.432 15.994-8.373 32.841-8.704 49.863H495.27z"/></g></svg>

```
No newline at end of file

File: packages/icons/src/Barcode.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...74-11.854 1.625-24.278 24.389-40.27 18.92-13.28 26.458-18.421 26.458-29.845 0-6.71-4.433-14.422-14.485-14.422-14.486 0-14.93 11.567-15.077 18.991h-20.102zm120.636 4.475h3.253c6.207 0 17.589-.57 17.589-13.71 0-3.711-1.775-12.992-13.156-12.992-14.19 0-14.19 12.993-14.19 16.563h-19.215c0-18.992 11.086-32.557 34.735-32.557 14.043 0 32.519 6.854 32.519 28.274 0 10.282-6.652 16.707-12.563 19.277v.286c3.695 1 15.815 6.283 15.815 23.562 0 19.136-14.634 32.7-35.77 32.7-9.904 0-36.509-2.141-36.509-33.843h20.25l-.148.144c0 5.424 1.477 17.706 15.372 17.706 8.277 0 15.52-5.428 15.52-16.707 0-13.853-13.894-14.423-23.502-14.423v-14.28zm141.182 38.65h-11.676v22.42h-20.693v-22.42h-41.388v-17.277l38.58-61.69h23.501v63.402h11.676v15.566zm-32.369-15.565c0-24.56.442-40.411.442-43.553h-.294l-26.31 43.553h26.162zm76.736-25.227c3.843-2.57 8.573-5.57 18.771-5.57 14.043 0 31.781 9.426 31.781 32.703 0 13.85-7.243 36.412-38.136 36.412-16.406 0-33.11-9.425-34.144-29.843h20.693c.887 8.28 5.47 13.849 14.634 13.849 9.903 0 15.669-8.139 15.669-18.99 0-8.998-4.436-18.136-15.375-18.136-2.807 0-9.902.142-13.893 7.283l-18.476-.858 7.243-55.691h57.646v16.85h-42.718l-3.695 21.99z" fill="#000" fill-rule="evenodd"/></svg>
```
Comment (LucasRuy) [inline packages/icons/src/Barcode.svg]: No new line at end of this file.
```suggestion
<svg width="512" height="512" xmlns="http://www.w3.org/2000/svg"><path d="M483.84 447.704V73.92h23.926v373.784h-23.925zm-483.84 0V73.92h23.925v373.784H0zm149.049-129.933V73.92h23.926v243.851H149.05zm110.863 0V73.92h23.926v243.851h-23.926zm37.926 0V73.92h23.925v243.851h-23.925zm111.477 0V73.92h23.927v243.851h-23.927zm37.57 0V73.92h23.927v243.851h-23.927zm-409.315 0V73.92h23.926v243.851H37.57zm37.57 0V73.92h60.374v243.851H75.14zm110.863 0V73.92h60.373v243.851h-60.373zm147.817 0V73.92h60.373v243.851H333.82zm-292.21 42.54h2.956c19.955 0 25.276-9.995 25.719-17.563h16.999v101.387H66.589v-69.829H41.61v-13.995zm69.495 18.278c0-32.986 24.98-35.84 34.292-35.84 20.102 0 36.065 11.422 36.065 31.986 0 17.42-11.676 26.275-21.876 32.985-13.894 9.425-21.136 13.995-23.501 18.709h45.525v17.706h-71.688c.74-11.854 1.625-24.278 24.389-40.27 18.92-13.28 26.458-18.421 26.458-29.845 0-6.71-4.433-14.422-14.485-14.422-14.486 0-14.93 11.567-15.077 18.991h-20.102zm120.636 4.475h3.253c6.207 0 17.589-.57 17.589-13.71 0-3.711-1.775-12.992-13.156-12.992-14.19 0-14.19 12.993-14.19 16.563h-19.215c0-18.992 11.086-32.557 34.735-32.557 14.043 0 32.519 6.854 32.519 28.274 0 10.282-6.652 16.707-12.563 19.277v.286c3.695 1 15.815 6.283 15.815 23.562 0 19.136-14.634 32.7-35.77 32.7-9.904 0-36.509-2.141-36.509-33.843h20.25l-.148.144c0 5.424 1.477 17.706 15.372 17.706 8.277 0 15.52-5.428 15.52-16.707 0-13.853-13.894-14.423-23.502-14.423v-14.28zm141.182 38.65h-11.676v22.42h-20.693v-22.42h-41.388v-17.277l38.58-61.69h23.501v63.402h11.676v15.566zm-32.369-15.565c0-24.56.442-40.411.442-43.553h-.294l-26.31 43.553h26.162zm76.736-25.227c3.843-2.57 8.573-5.57 18.771-5.57 14.043 0 31.781 9.426 31.781 32.703 0 13.85-7.243 36.412-38.136 36.412-16.406 0-33.11-9.425-34.144-29.843h20.693c.887 8.28 5.47 13.849 14.634 13.849 9.903 0 15.669-8.139 15.669-18.99 0-8.998-4.436-18.136-15.375-18.136-2.807 0-9.902.142-13.893 7.283l-18.476-.858 7.243-55.691h57.646v16.85h-42.718l-3.695 21.99z" fill="#000" fill-rule="evenodd"/></svg>

```

File: packages/icons/src/Magnifier.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg width="512" height="512" xmlns="http://www.w3.org/2000/svg"><path d="M199 36c-92.084 0-167 74.02-167 165 0 90.983 74.916 165 167 165s167-74.017 167-165c0-90.98-74.916-165-167-165m0 361C89.272 397 0 308.85 0 200.5S89.272 4 199 4s199 88.15 199 196.5S308.728 397 199 397zm306.834 73.766c8.221 8.15 8.221 21.486 0 29.636l-2.507 2.485c-8.221 8.15-21.675 8.15-29.897 0L338 368.634A228.427 228.427 0 0 0 369.887 336l135.947 134.766z" fill="#000" fill-rule="evenodd"/></svg>
```
Comment (LucasRuy) [inline packages/icons/src/Magnifier.svg]: No new line at end of this file.
```suggestion
<svg width="512" height="512" xmlns="http://www.w3.org/2000/svg"><path d="M199 36c-92.084 0-167 74.02-167 165 0 90.983 74.916 165 167 165s167-74.017 167-165c0-90.98-74.916-165-167-165m0 361C89.272 397 0 308.85 0 200.5S89.272 4 199 4s199 88.15 199 196.5S308.728 397 199 397zm306.834 73.766c8.221 8.15 8.221 21.486 0 29.636l-2.507 2.485c-8.221 8.15-21.675 8.15-29.897 0L338 368.634A228.427 228.427 0 0 0 369.887 336l135.947 134.766z" fill="#000" fill-rule="evenodd"/></svg>

```

File: packages/icons/src/Offers.svg [hunk 5]
```diff
@@ -0,0 +1 @@
...6 171.6 0 0 1-13.844 6.22l42.19 62.89a7.358 7.358 0 0 1 1.248 4.096v269.969c0 24.749-21.427 44.882-47.76 44.882H155.104c-12.558 0-25.492-4.596-35.48-12.61-10.896-8.739-16.898-20.2-16.898-32.272l-.126-269.585c0-1.506.461-2.977 1.322-4.212l115.26-165.124c5.124-8.105 13.951-12.939 23.66-12.939 9.732 0 18.582 4.858 23.699 13.004l18.972 28.28c13.146-13.889 34.608-34.012 57.55-45.016 14.948-7.168 51.695-21.236 70.113 2.894zM171.496 419.493l143.149-143.15a7.355 7.355 0 0 1 10.4 10.402l-143.15 143.15a7.327 7.327 0 0 1-5.2 2.155 7.355 7.355 0 0 1-5.2-12.557zm39.832-135.21c-9.947 0-18.043 8.093-18.043 18.043 0 9.947 8.096 18.041 18.043 18.041s18.043-8.094 18.043-18.041c0-9.95-8.096-18.044-18.043-18.044zm0 50.794c-18.06 0-32.751-14.693-32.751-32.751 0-18.06 14.691-32.753 32.751-32.753 18.06 0 32.751 14.693 32.751 32.753 0 18.058-14.691 32.75-32.75 32.75zm73.884 93.807c9.948 0 18.043-8.094 18.043-18.043 0-9.95-8.095-18.041-18.043-18.041-9.947 0-18.043 8.092-18.043 18.04 0 9.95 8.096 18.044 18.043 18.044zm0-50.794c18.06 0 32.751 14.691 32.751 32.75 0 18.06-14.691 32.754-32.75 32.754-18.06 0-32.752-14.693-32.752-32.753 0-18.06 14.691-32.751 32.751-32.751z" fill="#000" fill-rule="evenodd"/></svg>
```
Comment (LucasRuy) [inline packages/icons/src/Offers.svg]: No new line at end of this file.
```suggestion
<svg width="512" height="512" xmlns="http://www.w3.org/2000/svg"><path d="M392.788 71.24c13.277-15.32 17.098-35.941 8.695-46.946-8.35-10.944-27.327-10.42-52.056 1.443-16.714 8.017-36.789 24.075-55.496 44.304l32.62 48.623c31.989-12.1 52.216-31.249 66.237-47.424zm-152.474 73.769a7.355 7.355 0 0 1-6.983-7.71 7.325 7.325 0 0 1 7.71-6.981c7.01.347 13.696.404 20.132.25-2.438-7.806-9.733-13.487-18.33-13.487-10.586 0-19.197 8.612-19.197 19.198 0 10.583 8.611 19.196 19.197 19.196 7.323 0 13.696-4.124 16.932-10.17-1.713.033-3.429.062-5.161.062-4.698 0-9.466-.119-14.3-.358zm123.315 55.327l-43.06-64.19c-14.006 4.277-28.984 7.072-44.874 8.366-3.692 14.723-17 25.673-32.852 25.673-18.696 0-33.906-15.21-33.906-33.906 0-18.696 15.21-33.908 33.906-33.908 16.462 0 30.21 11.8 33.262 27.38 13.23-1.15 25.15-3.359 35.91-6.352L254.26 37.306a5.309 5.309 0 0 1-.14-.213c-2.403-3.867-6.62-6.176-11.278-6.176-4.66 0-8.878 2.31-11.28 6.176-.07.11-.142.219-.216.326L117.312 200.792l.125 267.27c0 15.514 18.306 30.178 37.668 30.178h175.473c18.224 0 33.051-13.537 33.051-30.173V200.336zm49.548-184.967c12.68 16.61 8.693 44.772-9.274 65.504-10.678 12.321-28.013 30.289-55.158 44.019a171.6 171.6 0 0 1-13.844 6.22l42.19 62.89a7.358 7.358 0 0 1 1.248 4.096v269.969c0 24.749-21.427 44.882-47.76 44.882H155.104c-12.558 0-25.492-4.596-35.48-12.61-10.896-8.739-16.898-20.2-16.898-32.272l-.126-269.585c0-1.506.461-2.977 1.322-4.212l115.26-165.124c5.124-8.105 13.951-12.939 23.66-12.939 9.732 0 18.582 4.858 23.699 13.004l18.972 28.28c13.146-13.889 34.608-34.012 57.55-45.016 14.948-7.168 51.695-21.236 70.113 2.894zM171.496 419.493l143.149-143.15a7.355 7.355 0 0 1 10.4 10.402l-143.15 143.15a7.327 7.327 0 0 1-5.2 2.155 7.355 7.355 0 0 1-5.2-12.557zm39.832-135.21c-9.947 0-18.043 8.093-18.043 18.043 0 9.947 8.096 18.041 18.043 18.041s18.043-8.094 18.043-18.041c0-9.95-8.096-18.044-18.043-18.044zm0 50.794c-18.06 0-32.751-14.693-32.751-32.751 0-18.06 14.691-32.753 32.751-32.753 18.06 0 32.751 14.693 32.751 32.753 0 18.058-14.691 32.75-32.75 32.75zm73.884 93.807c9.948 0 18.043-8.094 18.043-18.043 0-9.95-8.095-18.041-18.043-18.041-9.947 0-18.043 8.092-18.043 18.04 0 9.95 8.096 18.044 18.043 18.044zm0-50.794c18.06 0 32.751 14.691 32.751 32.75 0 18.06-14.691 32.754-32.75 32.754-18.06 0-32.752-14.693-32.752-32.753 0-18.06 14.691-32.751 32.751-32.751z" fill="#000" fill-rule="evenodd"/></svg>

```

**Pull request merged**